frontend/src/apiAxios.ts

```diff
@@ -11,7 +11,7 @@ export const setUserToken = (token: string) => {
 };
 
 export const getApiUrl = () => {
-  return "http://localhost:8080/api";
+  return "/api";
```

Comment:
This could use an env file, this was only hardcoded initially, but an env or use the same method that land use planning does with a default of localhost:xxxx and if in openshift uses the correct server.